### DISCUSSION TITLE
Why are there no icons of other social networks in the profile?

### DISCUSSION THREAD

**Discussion by rodukov** [community/community#57260] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/101352977/a7db25cb-f6bd-4886-8a4d-1809f92f3f90)
![image2](https://github.com/community/community/assets/101352977/f628d702-b76e-4302-8b4a-57ca22739f62)

🔺I am very glad that now github allows you to select 4 any other social networks, however I cannot understand why there is a separate icon for instagram, but not for discord.

Why these social networks?

**Comment by KAnejjar** [comment]
Some links are not included ...only links like linkedin,reddit,instagram,fb ..are included, so that's not actually an error ..

**Comment by datpmt** [comment, marked as answer]
## At the moment (2023-05-06), the following 9 social accounts seem to be supported.
- twitter
- facebook
- instagram
- linkedin
- reddit
- twitch
- youtube
- mastodon
- hometown
Read more in https://stackoverflow.com/a/76191105/20653131

**Reply by JakeSteam** [reply]
This is almost correct, only "npm" is missing.

**Reply by smo7x** [reply]
heyy jakesteam iam new to github idk shi8 about this but i want your help on something so please please contact me on instagram @_pr9v or reply HERE PLEASE
